### ISSUE TITLE
tech(wmg): update WMG snapshot schemas to comment out assay and development ontology term IDs

### ISSUE THREAD

**Issue by atarashansky** [chanzuckerberg/single-cell-data-portal#5306]
Including unused metadata in the snapshots are making them unnecessarily large. We can shrink them by a factor of roughly 30-40% if we comment out assay and development ontology term IDs from the pipeline.

Any future work that requires this metadata can be easily enabled by uncommenting them in the schema. Doing this would improve performance in the short term and not prevent us from supporting features that require this metadata in the future.

**Comment by kaloster** [comment]
closing this as per @atarashansky

**Comment by atarashansky** [comment]
Reopening - it's not officially complete it. Will close once the new WMG pipeline is merged.
